### DISCUSSION TITLE
Getting errors when switching windows while using blender (xcb error: op ChangeProperty (no minor))

### DISCUSSION THREAD

**Discussion by charbelnicolas** [labwc/labwc#1502] [Q&A]
Hello, any help is appreciated... 

When using blender and switching focus from the blender window to another window I sometimes get xcb errors... I don't know if this is related to a bug which makes blender unable to receive any input from the mouse on the current viewport after switching a window with the window switcher. Apparently this happens on SwayWM too per the blender issue tracker.

[https://projects.blender.org/blender/blender/issues/117225#issuecomment-1114837](https://projects.blender.org/blender/blender/issues/117225#issuecomment-1114837)

Here's the screenshot of the errors I get while using blender:

![ss_2024_02_03_17_32_48](https://github.com/labwc/labwc/assets/14960414/37308058-044b-490f-b809-32dad229034c)

Is this something that has to do with wlroots or labwc? Or is this something on my part?

Thank you

**Comment by charbelnicolas** [comment]
I haven't tested other wlroots compositors myself but since the blender issue states that swaywm suffers from this as well then it must be an issue with wlroots?

**Comment by Consolatis** [comment]
Continued at
- #1507
